alembic/versions/9575e8399c9f_create_gardens.py

```diff
@@ -0,0 +1,31 @@
+"""create gardens
+
+Revision ID: 9575e8399c9f
+Revises: 752c338c6328
+Create Date: 2020-07-23 20:37:51.139055
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from datetime import datetime
+
+
+# revision identifiers, used by Alembic.
+revision = '9575e8399c9f'
+down_revision = '752c338c6328'
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    op.create_table(
```

Comment:
Nice work on these Alembic migrations.  We'll need to add one or two fields to this in the future but a great start!